src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
+  --border: 2px;
+  --active-color: #00acdc;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+img {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  list-style: none;
+  margin: 0 10px;
+}
+
+.nav__link {
+  color: black;
+  display: flex;
+  position: relative;
+  height: 60px;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  text-decoration: none;
+  align-items: center;
+}
+
+.nav__item:first-child {
+  margin: 0 10px 0 0;
+}
+
+.nav__item:last-child {
+  margin: 0 0 0 10px;
+}
+
+.nav__link:visited {
+  color: black;
+}
+
+.nav__link.is-active {
+  color: var(--active-color);
+}
+
+.nav__link:hover {
+  color: var(--active-color);
+}
+
+.is-active::after {
+  position: absolute;
+  top: calc(100% - (var(--border) * 2));
+  width: calc(100% - (var(--border) * 2));
+  content: "";
+  border: solid var(--border) var(--active-color);
+  border-radius: 8px;
```

Comment:
Your calculation looks too complicated, we can do the same in easier way
![image](https://github.com/mate-academy/layout_moyo-header/assets/80700247/745a7161-1ca8-45e2-b9c6-06fac70e16fb)

Reply:
problem only in calculation?